### PR TITLE
test: add storybook stories for public pages

### PR DESCRIPTION
**Pull request by codeincontext**
Adds storybook stories for each of our public pages

In the long run we might want more targeted stories for the atoms and molecules which make up pages, but for now this is a good way to get test coverage

For each page I've created an inner `MyPageContent` component without the `<Layout>` wrapper. We want to test the layout independently. This has caused large diffs due to the indentation change, but there actually isn't that much being changed

### CODE REVIEW

File: apps/nextjs/src/app/faqs/index.tsx [hunk 1]
```diff
@@ -36,821 +36,816 @@ const FAQPage = () => {
     }
   };
   return (
-    <Layout>
```
Comment (codeincontext) [inline apps/nextjs/src/app/faqs/index.tsx:39]: The only change here is removing this `<Layout>`

File: apps/nextjs/src/cms/types/policyDocument.ts [hunk 2]
```diff
@@ -1,7 +1,7 @@
 export interface PolicyDocument {
   title: string;
   slug: string;
-  fake_updatedAt: string;
+  fake_updatedAt: string | null;
```
Comment (codeincontext) [inline apps/nextjs/src/cms/types/policyDocument.ts:4]: The fixture data was failing the type

**Review by JBR90 — APPROVED** [review]
Looks good!

**Review by mikeritson-oak — APPROVED** [review]
This looks good. Checked in Chromatic and approved and, as this is a new process, also checked that all of the affected pages still render in the preview branch. 👍

**Pull request merged**